### DISCUSSION TITLE
Stop code snippets from inserting automatically

### DISCUSSION THREAD

**Discussion by apfel-dev** [neovim/neovim#25618] [General]
I am just starting out using neovim v0.9.1 and using [kickstrat.nvim](https://github.com/nvim-lua/kickstart.nvim) and I love it.

I just have one small complain: when writing code (especially for bash) if I type something that matches a snippet, nvim will insert it when I press enter. For example the typing in `fi + <enter>` for the closing statement of an if statement in a bash script inserts the bash for loop code snippet.
Is there a way I can get finer control over when snippets are inserted (for example by using the arrow keys) ? At the moment I have to go back to insert mode and then `o` to start on the next line.

**Comment by clason** [comment]
That depends completely on your snippet plugin, Neovim itself does not provide any snippet functionality.

**Reply by apfel-dev** [reply]
So that means I posted in the wrong forum. Sorry
